tools/rebasehelpers/commitchecker/commitchecker.go

```diff
@@ -36,8 +36,13 @@ func main() {
 
 	errs := []string{}
 	for _, validate := range AllValidators {
-		err := validate(nonbumpCommits)
-		if err != nil {
+		if err := validate(nonbumpCommits); err != nil {
+			errs = append(errs, err.Error())
+		}
+	}
+	if len(os.Getenv("GODEP_RESTORE")) > 0 {
```

Comment:
The only reason I'm not enabling this always is because commitchecker is run inside travis and I doubt it'll succeed there. The env is always on in `make check`, which is always run in jenkins. Then it looks at either modified `Godeps/_workspace`, `vendor/` dirs or `Godeps/Godeps.json`, if any of these is touched it'll run `hack/godep-restore.sh`.